scripts/install-lansa-base.ps1

```diff
@@ -98,6 +98,7 @@ try
     if ( !(test-path $TempPath) ) {
         New-Item $TempPath -type directory -ErrorAction SilentlyContinue | Out-Default | Write-Host
     }
+    #  Enabling TLS 1.2 security protocol to establsih a secure connection with the server when making web requests.
     [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```

Comment:
Somebody else has added this code somewhere. Fin d out who it is and in what code and I will decide what is to be done.

Reply:
Yes Robert correct, I and Mohit both have similar changes so I am following #192 PR and looking for your comments over there, so one way what I think to resolve it would be as both of us have run all builds and they are successful we can let Mohit PR we the main source of code and once you review it and merge it maybe I can later pull the code and run all builds again for verification.
Do let me know if that is okay.

Reply:
ok